rclrs/src/qos.rs

```diff
@@ -178,41 +183,47 @@ impl From<QoSDurabilityPolicy> for rmw_qos_durability_policy_t {
     }
 }
 
+/// Equivalent to `rmw_qos_profile_sensor_data`.
```

Comment:
It might be useful to link to `rmw_qos_profile_sensor_data` from `rmw`

Reply:
Which link do you have in mind? GitHub master, GitHub pinned revision, https://docs.ros2.org/galactic/api/rmw/qos__profiles_8h_source.html?

Reply:
I'd use master, what do you think?

Reply:
Ok. I added a link to the `qos_profiles.h` file to the docstring, but didn't hyperlink `rmw_qos_profile_sensor_data` directly, because the specific line where it's defined in might change.